falco/client.py

```diff
@@ -2,23 +2,38 @@
 
 from falco.client_credentials import get_grpc_channel_credentials
 from falco.domain import Response
-from falco.svc.output_pb2_grpc import serviceStub
+from falco.svc.outputs_pb2_grpc import serviceStub
 
 
 class InvalidFormat(Exception):
     pass
 
+class TLSConfigError(Exception):
+    print(Exception)
+
 
 class Client:
-    def __init__(self, endpoint, client_crt, client_key, ca_root, output_format=None, *args, **kw):
-        self._client = serviceStub(
-            grpc.secure_channel(
-                endpoint,
-                credentials=get_grpc_channel_credentials(client_crt, client_key, ca_root),
-                options=[("grpc.max_receive_message_length", 1024 * 1024 * 512)],
-            ),
-        )
-        self.output_format = output_format
+    def __init__(self, endpoint, client_crt=None, client_key=None, ca_root=None, output_format=None, *args, **kw):
+        if endpoint.startswith("unix:///"):
+            self._client = serviceStub(
+                grpc.insecure_channel(
```

Comment:
👍 

I was trying to use `secure_channel` + [local channel credentials](https://grpc.github.io/grpc/python/grpc.html#grpc.local_channel_credentials) but it didn't work for some reason

Reply:
I'm a bit of a n00b at grpc. Do we have to do something on the falco/server side to make that work?

Reply:
FWIW the go-client seems to use insecure for the UDS connection https://github.com/falcosecurity/client-go/blob/master/pkg/client/client.go#L49

Reply:
I would leave it as it is now (insecure_channel), I did some tests this weekend and I could't find a way to make it work with the socket opened by the falco server. Also, `This is an EXPERIMENTAL API.` is not encouraging :D

Reply:
XD ok. Maybe soon!